tests/console/check_system_info.pm

```diff
@@ -23,29 +23,29 @@ use strict;
 use warnings;
 use testapi;
 use utils;
-use version_utils 'is_sle';
+use version_utils;
 use registration;
 use List::MoreUtils 'uniq';
 
 sub run {
     select_console('root-console');
     assert_script_run('setterm -blank 0') unless (check_var('ARCH', 's390x'));
 
-    assert_script_run("SUSEConnect --status-text > /dev/$serialdev", timeout => 180);
-
     if (!get_var('MILESTONE_VERSION')) {
         assert_script_run('cat /etc/issue');
     } else {
         my $milestone_version = get_var('MILESTONE_VERSION');
         assert_script_run("grep -w $milestone_version /etc/issue");
     }
-    assert_script_run('cat /etc/os-release');
+
+    script_run('zypper lr | tee /tmp/zypperlr.txt');
 
     my $myaddons = get_var('SCC_ADDONS', "");
-    $myaddons = $myaddons . ",base,serverapp"                             if (is_sle('15+') && check_var('SLE_PRODUCT', 'sles'));
-    $myaddons = $myaddons . ",base,desktop,we,python2"                    if (is_sle('15+') && check_var('SLE_PRODUCT', 'sled'));
-    $myaddons = $myaddons . ",base,serverapp,desktop,dev,lgm,python2,wsm" if (is_sle('<15', get_var('ORIGIN_SYSTEM_VERSION')) && is_sle('15+'));
-    $myaddons = $myaddons . ",python2"                                    if (is_sle('=15', get_var('ORIGIN_SYSTEM_VERSION')) && is_sle('15+'));
+    $myaddons = $myaddons . ",base,serverapp"                                  if (is_sle('15+') && check_var('SLE_PRODUCT', 'sles'));
```

Comment:
You can use .= to make it more easy.

Reply:
Besides, I think this line can't distinguish the SLE12 or SLE15 since this module is loaded after migration and is_sle will only judge the 'VERSION' which is the target version and now is 15SP3.

Reply:
Besides, I think this line can't distinguish the SLE12 or SLE15 since this module is loaded after migration and is_sle will only judge the 'VERSION' which is the target version and now is 15SP3.